src/mainboard/clevo/adl-p/devicetree.cb

```diff
@@ -20,6 +20,13 @@ chip soc/intel/alderlake
 	# Thermal
 	register "tcc_offset" = "8"
 
+	# Acoustic settings
+	register "acoustic_noise_mitigation" = "1"
+	register "slow_slew_rate[VR_DOMAIN_IA]" = "SLEW_FAST_8"
+	register "slow_slew_rate[VR_DOMAIN_GT]" = "SLEW_FAST_8"
+	register "fast_pkg_c_ramp_disable[VR_DOMAIN_IA]" = "1"
+	register "fast_pkg_c_ramp_disable[VR_DOMAIN_GT]" = "1"
```

Comment:
Is the noise reduced with these settings? How did you determine the settings?

Reply:
Removed in the latest patchset.